BitwardenShared/UI/Vault/Vault/AutofillList/VaultAutofillListProcessor.swift

```diff
@@ -358,7 +358,6 @@ extension VaultAutofillListProcessor {
             }
         case let .autofillFido2VaultList(serviceIdentifiers, fido2RequestParameters):
             state.isAutofillingFido2List = true
-            state.emptyViewMessage = Localizations.noItemsToList
```

Comment:
🤔 Is this no longer necessary?

Reply:
Setting `emptyViewMessage`? I don't think so, it will default to `There are no items in your vault for this website/app. Tap to add one.`, which I think is what we would want to say in this scenario?

https://github.com/bitwarden/ios/blob/4de2663e2b7d9c598aacdb220d61ee95c3c914f4/BitwardenShared/UI/Vault/Vault/AutofillList/VaultAutofillListState.swift#L15

Reply:
Oh you're right with the change in behavior of the add button, now this is no longer necessary. Thanks for checking it!